client/gefyra/cli/utils.py

```diff
@@ -237,6 +238,42 @@ def parse_workload(ctx, param, workload: str) -> str:
     return workload
 
 
+class FileFromArgument(NamedTuple):
+    workload: str
+    source: str
+    destination: str = "/"
+
+
+def parse_file_from(ctx, param, file_from_arguments) -> Tuple[FileFromArgument]:
+    if not file_from_arguments:
+        return None
+
+    MSG = (
+        "Invalid argument format. Please provide the file "
+        "in the format 'type/name:source:destination' or "
+        "'type/name/container-name:source:destination'."
```

Comment:
Could we make <destination> optional? (I would suspect in most of the cases users will want the file to land in the same place it was copied from.)

Reply:
Yes, I agree. Good point.